tests/unit/specs/store/connection.spec.js

```diff
@@ -138,6 +138,13 @@ describe(`Module: Connection`, () => {
           address_prefix: "cosmos",
           network_type: "cosmos",
           testnet: true
+        },
+        {
+          id: "terra-testnet",
```

Comment:
should include e-money here too, no? and terra-mainnet?

Reply:
this is a unit test nor our whole networks list